Nos/Views/NoteCard.swift

```diff
@@ -96,14 +78,11 @@ struct NoteCard: View {
                         }
                         BeveledSeparator()
                         HStack(spacing: 0) {
-                            if showReplyCount {
-                                StackedAvatarsView(avatarUrls: replyAvatarURLs, size: 20, border: 0)
-                                    .padding(.trailing, 8)
-                                if let replies = attributedReplies {
-                                    Text(replies)
-                                        .font(.clarity(.medium, textStyle: .subheadline))
-                                        .foregroundColor(Color.secondaryTxt)
-                                }
+                            if repliesDisplayType != .displayNothing {
+                                RepliesLabel(
+                                    repliesDisplayType: repliesDisplayType,
+                                    for: note
+                                )
```

Comment:
I get why we might want to check repliesDisplayType first, but couldn't RepliesLabel just return `EmptyView` if it's `.displayNothing`? This could would read more nicely for sure:

```suggestion
                            RepliesLabel(
                                repliesDisplayType: repliesDisplayType,
                                for: note
                            )
```

Reply:
RepliesLabel already returns an `EmptyView`, I had to add that after receiving the feedback of stop using an optional but adding a .none case (that I had to change to .displayNothing because .none is already implemented in Swift. So far so good, but I don't like to do what you suggest here. Right now, the spacing of the HStack is zero, but if it were different than zero, having a view that the implementation contains an EmptyView adds an extra spacing. So, as a general rule, either communicate that with an optional, or do the checking twice, here and in RepliesLabel.